installer/docker/README.md

```diff
@@ -1,5 +1,125 @@
 # HNN Docker Container
 
+### ** UPDATED Instructions: 11/29/2023 **
+
+### Windows Installation Process
+
+## Part 1 Install WSL
```

Comment:
Is this weird heading ordering intentional? It looks like you're trying to highlight that there are updated instructions, and then you're looking at windows, and the first step of that is to install WSL. The heading ordering might make this less valuable for people who use a screenreader, which relies on H2 being above H3.

```suggestion
** UPDATED Instructions: 11/29/2023 **

## Windows Installation Process

### Part 1 Install WSL
```